chain/chain/src/resharding.rs

```diff
@@ -362,7 +362,7 @@ mod tests {
             state_root = new_state_root;
 
             // update split states
-            let trie_changes = tries
+            let mut trie_updates = tries
```

Comment:
Sus, this seems a bit similar to changes from a PR by Shreyan...

Reply:
Didn't touch that file. Rebased and this change disappeared.